src/aind_metadata_service/response_handler.py

```diff
@@ -87,116 +139,99 @@ def model_response(model: BaseModel) -> JSONResponse:
-            if sp_status_code < 300 and lb_status_code < 300:
-                lb_data = lb_contents["data"]["subject_procedures"]
-                sp_data = sp_contents["data"]["subject_procedures"]
-                combined_data = lb_data + sp_data
-                lb_contents["data"]["subject_procedures"] = combined_data
-                return JSONResponse(
-                    status_code=StatusCodes.VALID_DATA.value,
-                    content=(
-                        {
-                            "message": "Valid Model.",
-                            "data": lb_contents["data"],
-                        }
-                    ),
-                )
-            # handles combination of server/connection error and valid response
-            if sp_status_code in (500, 503) and lb_status_code < 300:
-                lb_data = lb_contents["data"]["subject_procedures"]
-                lb_contents["data"]["subject_procedures"] = lb_data
-                return Responses.multi_status_response(
-                    message1=sp_contents["message"],
-                    message2=lb_contents["message"],
-                    data=lb_contents["data"],
-                )
-            if sp_status_code < 300 and lb_status_code in (500, 503):
-                sp_data = sp_contents["data"]["subject_procedures"]
-                sp_contents["data"]["subject_procedures"] = sp_data
-                return Responses.multi_status_response(
-                    message1=sp_contents["message"],
-                    message2=lb_contents["message"],
-                    data=sp_contents["data"],
-                )
-        except Exception as e:
-            logging.error(repr(e))
-            return Responses.internal_server_error_response()
+        lb_model = lb_response[1]
+        sp_model = sp_response[1]
+        sp_status_code = sp_response[0]
+        lb_status_code = lb_response[0]
+        if sp_status_code == 500 and lb_status_code == 500:
+            message = Responses.generate_message(sp_status_code)
+            return Responses.internal_server_error_response() + (message,)
+        if sp_status_code == 503 and lb_status_code == 503:
+            message = Responses.generate_message(sp_status_code)
+            return Responses.connection_error_response() + (message,)
+        if sp_status_code == 404 and lb_status_code == 404:
+            message = Responses.generate_message(sp_status_code)
+            return Responses.no_data_found_response() + (message,)
+        if sp_status_code < 300 and lb_status_code == 404:
+            message = Responses.generate_message(sp_status_code)
+            return sp_response + (message,)
+        if sp_status_code == 404 and lb_status_code < 300:
+            message = Responses.generate_message(lb_status_code)
+            return lb_response + (message,)
+        # handles combination of invalid responses
+        if lb_status_code == 406 and sp_status_code == 406:
+            print("both invalid")
```

Comment:
We haven't setup a logger yet, but we don't want to use `print`. You can remove this line